src/js/config/Config.ts

```diff
@@ -56,7 +56,7 @@ let Config: IConfiguration = {
   historyLength: 31,
   historyServer: "",
   fullProductName: "Mesosphere DC/OS",
-  marathonAPIPrefix: "/service/marathon/v2",
+  marathonAPIPrefix: global.MARATHON_API_PREFIX,
```

Comment:
@pierrebeitz maybe you could help me silencing TS lint. It complains `MARATHON_API_PREFIX` is not defined on global #no_pressure

Reply:
having discussed this exact issue a lot in the past, i'd propose to add a special file to augment builtins. this way there's a single place to look up the "imperfections" rather than relying on `(global as any).SOME_BLORGH` or extending those modules on a per-file basis.

it could look something like the following (although i did not test this).
```
// e.g. src/typings/global.d.ts
declare namespace NodeJS {
  interface Global {
    MARATHON_API_PREFIX: string;
  }
}
```

Reply:
Thanks! Had to adjust tslint config for that to work 😬